src/Components/Register/Register.js

```diff
@@ -21,59 +21,155 @@ const Register = ({ onSubmit, errorLabel }) => {
     password: '',
     repeatPassword: '',
     alterEgo: '',
-    icon: '' });
+    icon: '',
+    isPwdMatched: false
+  });
 
   const handleChange = ({ target }) => {
-    const { name, value } = target;
+
+  const { name, value } = target;
+
+  let checkPwdValidity = {};
+    if ( name === 'password' ) {
+        if ( value === payload.repeatPassword && value !== '' ) {
+          checkPwdValidity.isPwdMatched = true;
+        } else {
+          checkPwdValidity.isPwdMatched = false;
+        };
+      };
+    if ( name === 'repeatPassword' ) {
+        if ( value === payload.password && value !== '' ) {
+          checkPwdValidity.isPwdMatched = true;
+        } else {
+          checkPwdValidity.isPwdMatched = false;
+        };
+      };
     setPayload({
       ...payload,
+      ...checkPwdValidity,
       [name]: value
     });
   };
 
   const handleSubmit = e => {
     e.preventDefault();
-    onSubmit(payload);
+    if ( payload.isPwdMatched ) {
+      onSubmit( payload );
+    } else {
+      alert( "Check password" )
+    };
   };
 
   const { email,
     userName,
     password,
     repeatPassword,
-    alterEgo } = payload,
-  { language }          = config,
-  translations          = getTranslations(language), //ToDo: Replace this language for config directly or from store
-  registerTranslations  = translations.register,
-  titleLabel            = registerTranslations.title,
-  emailLabel            = registerTranslations.email,
-  userNameLabel         = registerTranslations.userName,
-  passwordLabel         = registerTranslations.password,
-  repeatPasswordLabel   = registerTranslations.repeatPassword,
-  alterEgoLabel         = registerTranslations.alterEgo,
-  iconLabel             = registerTranslations.icon,
-  submitLabel           = registerTranslations.submit;
+    alterEgo,
+    isPwdMatched
+  } = payload,
+  { language, passwordSettings }          = config,
```

Comment:
Please reduce the indentation after the =

Reply:
Done.